gax-bom/pom.xml

```diff
@@ -65,4 +72,16 @@
       </dependency>
     </dependencies>
   </dependencyManagement>
+
+  <build>
```

Comment:
We probably don't this checkstyle plugin in the bom but is it causing any trouble leaving it there?

Reply:
I was facing this error:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.2.0:check (checkstyle) on project gax-bom: Failed during checkstyle execution: Unable to find configuration file at location: license-checks.xml: Could not find resource 'license-checks.xml'. -> [Help 1]
```

But, on second thought it seems odd the monorepo doesn't have this issue for the boms. Taking another look.

Reply:
Adding in this config fixes the issue: `<configLocation>${project.basedir}/../license-checks.xml</configLocation>`. Seems I had to specify the license-check.xml and java.header file relative to the bom's pom.xml. Seems unnecessary to have, so I'd rather just keep the <skip>true</skip> in the bom as we move into the monorepo.